apps/bfd-server/bfd-server-war/pom.xml

```diff
@@ -219,15 +219,6 @@
 			<version>3.10.0</version>
 			<scope>test</scope>
 		</dependency>
-		<!-- Hibernate brings in an older version of byte-buddy which seems
-		to conflict with mockito, which wants version 1.11.0.  Bringing it
-		in as a first class dependency to resolve the issue. -->
-		<dependency>
-			<groupId>net.bytebuddy</groupId>
-			<artifactId>byte-buddy</artifactId>
-			<version>1.11.0</version>
-			<scope>test</scope>
-		</dependency>
```

Comment:
Latest version of hibernate does not have this issue. Leaving this here with the newer version of hibernate leads to other issues so removing this now unnecessary workaround.